pyproject.toml

```diff
@@ -256,23 +261,26 @@ extend-exclude = [
 [tool.poe.tasks]
 
 check = [
-  # "lint",
+  "lint",
   "test",
 ]
 
 format = [
-  { cmd = "black ." },
-  # Configure Ruff not to auto-fix (remove!):
-  # unused imports (F401), unused variables (F841), `print` statements (T201), and commented-out code (ERA001).
-  { cmd = "ruff --fix --ignore=ERA --ignore=F401 --ignore=F841 --ignore=T20 --ignore=ERA001 ." },
+  # Format project metadata.
   { cmd = "pyproject-fmt --keep-full-version pyproject.toml" },
+
+  # Format code.
+  # Configure Ruff not to auto-fix a few items that are useful in workbench mode.
+  # e.g.: unused imports (F401), unused variables (F841), `print` statements (T201), commented-out code (ERA001)
+  { cmd = "ruff format" },
+  { cmd = "ruff check --fix --ignore=ERA --ignore=F401 --ignore=F841 --ignore=T20 --ignore=ERA001" },
 ]
 
 lint = [
-  { cmd = "ruff ." },
-  { cmd = "black --check ." },
+  { cmd = "ruff format --check" },
+  { cmd = "ruff check" },
   { cmd = "validate-pyproject pyproject.toml" },
-  { cmd = "mypy" },
+  # { cmd = "mypy" },
```

Comment:
Adjusting the code base for type-safety did not happen within this patch, because I did not want to mix up concerns too much, by adding those changes within the noise of code-formatting only.

It will be tackled on a subsequent iteration.